reconcile/openshift_serviceaccount_tokens.py

```diff
@@ -16,21 +15,33 @@
 QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
 
 
-def construct_sa_token_oc_resource(name, sa_name, sa_token):
+def construct_sa_token_oc_resource(name, sa_token):
     body = {
         "apiVersion": "v1",
         "kind": "Secret",
         "type": "Opaque",
         "metadata": {
             "name": name,
         },
-        "data": {"token": base64.b64encode(sa_token).decode("utf-8")},
+        "data": {"token": sa_token},
     }
     return OR(
         body, QONTRACT_INTEGRATION, QONTRACT_INTEGRATION_VERSION, error_details=name
     )
 
 
+def get_token_for_service_account(service_account, tokens):
+    result = []
```

Comment:
why returning a list and not the first matching token found ?

Reply:
I originally used a lambda but was getting a lint error "Cell variable sa_name defined in loop" so pulled that part out as a function. Now looking at it yes that would be a better way to write this function.